syndesis/plugins/org.fusesource.ide.syndesis.extension.core/src/org/fusesource/ide/syndesis/extensions/core/util/SyndesisExtensionsUtil.java

```diff
@@ -42,4 +52,87 @@ public static boolean isValidSyndesisExtensionVersion(String version) {
 		}
 		return valid;
 	}
+	
+	/**
+	 * retrieves important version information from an online version mapping file
+	 * 
+	 * @param urlString
+	 * @return
+	 * @throws IOException
+	 */
+	public static IgniteVersionInfoModel getIgniteVersionModel(String urlString) {
+		IgniteVersionInfoModel model = new IgniteVersionInfoModel();
+		Properties vMapping = new Properties();
+		
+		try {
+			URL url = new URL(urlString);
+			vMapping.load(url.openStream());
+		} catch (IOException ex) {
+			// we ignore load errors
```

Comment:
why ignoring them?
Might worth reporting them, it will be a notification that we removed the remote file although we shouldn't

Reply:
method is used in a unit test. if we log the error via activator plugin log then we need to create integration test instead.

Reply:
I think that it was the case "only because" the url used was on master and it was not merged yet.
I created https://issues.jboss.org/browse/FUSETOOLS-2768 to improve it